package.json

```diff
@@ -147,7 +147,8 @@
     "build:test": "npm-run-all test linters",
     "build:release": "npm-run-all build build:test",
     "build:ci": "npm-run-all sweep build:release",
-    "build:api": "node scripts/wca.cjs 'src/auro-flight*.js' --outFiles docs/api.md",
+    "build:api": "wca 'docs/wca/*' --outFiles docs/api.md",
+    "build:api:prepare": "node scripts/wca.cjs 'src/auro-flight*.js'",
```

Comment:
I'm not following why we need this script and the associated files given the script on the line above (150) and the new files under `docs/wca`.